src/server/twemcache/main.c

```diff
@@ -1,3 +1,4 @@
+#include "admin/process.h"
```

Comment:
This looks out of place... why are you changing this file now, here, in this branch?